doc/Architecture/spec-al_x-road_audit_log_events.md

```diff
@@ -0,0 +1,465 @@
+# Audit log events
+
+Version: 1.9
+16.02.2023
+Doc. ID: SPEC-AL
```

Comment:
A line break seems to be missing from here in the `html` version.

Reply:
Thanks, fixed. Also removed the date field, as we don't seem to use it in other documents anymore either.